.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/comit
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "Cargo.lock" }}
+      - setup_system
+      - setup_nodejs
+      - setup_rust
+      - print_current_versions
+      - run:
+          name: Build using cargo make
+          command: |
+            source $HOME/.cargo/env
+            cargo --version
+            cargo make ci
+      - save_cache:
+          key: dependency-cache-{{ checksum "Cargo.lock" }}
+          paths:
+            - "~/.cargo"
+            - "./target"
```

Comment:
Why not cache the `sccache` folder like in travis?
What about `node_modules`?

Reply:
❓ why caching `./target`? We did not use to do that for Travis.

Reply:
@thomaseizinger : 
yes, should cache `node_modules` as well
sccache: I don't think we need it, at least with the current setup.

@D4nte :
we might not need to build everything all the time, hence I cache ./target